dependencies/FindFastJet.cmake

```diff
@@ -107,7 +107,21 @@ if(${${PKGNAME}_FOUND})
   foreach(incdir ${${PKGNAME}_INCLUDE_DIRS})
     target_include_directories(${PKGNAME}::${PKGNAME} INTERFACE ${incdir})
   endforeach()
+
```

Comment:
I would prefer to keep the modification of the FastJet-related cmake stuff to a separate PR, mostly to keep track of what's going on and potentially react in case of issues.

Reply:
Sorry yes i forgot to make it separate. Would you like to push it and then I can remove it from this pull request

Reply:
this is now removed, so if you could add it to your pull request that would be great